src/main/resources/pdiresources/transformations/merge-location.ktr

```diff
@@ -1,823 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
why have you decided to get rid of this please?

Reply:
This was added by mistake. The transformation was not complete, but file is still available in the corresponding ticket.